resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,30 @@
+<idea-plugin url="https://github.com/strawberry-graphql/strawberry-pycharm-plugin" require-restart="true">
+    <id>rocks.strawberry</id>
+    <name>Strawberry-Graphql</name>
```

Comment:
```suggestion
    <name>Strawberry-GraphQL</name>
```

where is this name used? :)

Reply:
Here!!
https://plugins.jetbrains.com/plugin/12861-pydantic